.github/workflows/notify-discord.yml

```diff
@@ -0,0 +1,25 @@
+name: Notify Discord
+
+on:
+  push:
+    tags:
+      - '@faustwp/wordpress-plugin@*'
+      - '@faustwp/core@*'
+      - '@faustwp/cli@*'
+      - '@faustwp/blocks@*'
+      # Test this action without creating a full release.
+      - '@faustwp/test@*'
+      '@faustwp/test-for-pr@*'
+
+jobs:
+  release_plugin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Discord Notification
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+          DISCORD_USERNAME: Faust.js Bot
+          DISCORD_AVATAR: https://pbs.twimg.com/profile_images/1658588429385601025/QZhc7IAR_400x400.png
+        uses: Ilshidur/action-discord@0c4b27844ba47cb1c7bee539c8eead5284ce9fa9
+        with:
+          args: '${{ github.ref_name }} has been released!'
```

Comment:
Could we link to the release in GitHub as well? Or include the changes?

Reply:
Love the idea of linking and/or including changelog.

Reply:
Thanks y'all! Thoughts on this?

<img width="437" alt="Screenshot 2023-06-05 at 10 46 24 PM" src="https://github.com/wpengine/faustjs/assets/6676674/2dad4061-2255-40e0-a937-8ff56a8e7974">

Link to test message https://discord.com/channels/836253505944813629/1114240986755911730/1115471414120693802